### DISCUSSION TITLE
Secrets for push services in openshift deployment templates

### DISCUSSION THREAD

**Discussion by mbastian93** [NUMde/compass-numapp-backend#41] [General]
Since #21 the firebase cloud messaging service is used for push notification, therefore the setup/deployment process for openshift has slightly changed. I have been wondering, if the parameters

  - PUSH_CLIENT_SECRET
  - PUSH_API_KEY
  - PUSH_APP_ID

are still relevant? As far as I can tell, all the information now comes from google-services.json.

@alenaSchmickl @lenzch

**Comment by lenzch** [comment]
Indeed, the parameters are not needed anymore.
